### DISCUSSION TITLE
cannot get a vim.cmd([[ highlight LineNr guibg=#002240 ]]) to work

### DISCUSSION THREAD

**Discussion by 11raob** [NvChad/NvChad#2758] [Q&A] [answered]
I have tried three different ways to get this command to work inside ~/.config/nvim/lua/custom/init.lua or in ~/.config/nvim/after/user_defined.lua.  The command just changes the foreground and background colors of the line number column.

The command is
```
vim.command([[
  highlight LineNr guibg=#002240
  highlight LineNr guifg=#e2c3db
]])
```

The darnedest part is that I can manually get the colors by typing (in Normal mode)
```
:highlight LineNr guibg=#002240  # and 
:highlight LineNr guifg=#e2c3db
```

If there were a way for me to write out these manual changes so that they would persist, that would be great.

I tried also writing a function and then calling the function, and I also tried using the vim api for setting LineNr foreground and background, but neither of those two approaches worked.  I've even tried it out on another machine where I have NvChad support layer added to Neovim, and I cannot get this to work on that machine either (one is NixOS; the other is macOS).  Otherwise, the NvChad is working perfectly with my other plugins and configurations:  treesitter, lsp, etc.  

Do I have some conflict between what plugsins I have installed and this vim.cmd command?  Why does it work at the input line with the :highlight ... manual setting, but not when I include the instructions in an init.lua file?

**Comment by siduck** [comment, marked as answer]
https://nvchad.com/docs/config/theming/

use hl_add for new hlgroups, hl_override for existing i.e modified by nvchad

**Reply by 11raob** [reply]
Thank you, @siduck 
I ended up with
```
M.ui = {
  theme = 'ayu_dark',
  hl_override = {

    ["LineNr"] = {fg = "#e2c3db", bg = "#002240"}

  }
}
```
It works perfectly.  Next, I just need to get the file "explorer" / tree showing automatcially upon launch instead of having to type Ctrl-n always to toggle it on / off.

**Reply by siduck** [reply]
```bash

nv() {
  [ -n "$1" ] && [ -d "$1" ] && cd "$1" && shift
  nvim +'lua require("nvim-tree.api").tree.open()'  "$@"
}
```

@11raob  or use an autocmd like at events Verylazy or vimenter to run nvimtree command

**Reply by 11raob** [reply]
Sweet!  You are awesome, man.  Brother, what a help you are!  I could not have come up with that function.  It's just exactly what I wanted, though.  I am so appreciative of your time and concern to help me.
